cmd/gaia/cmd/gaiad/main.go

```diff
@@ -23,9 +23,9 @@ import (
 )
 
 // gaiad custom flags
-const flagAssertInvariantsBlockly = "assert-invariants-blockly"
+const flagInvariantsChecksFrequency = "inv-checks-freq"
```

Comment:
```suggestion
const flagInvarCheckFreq = "inv-checks-freq"
```

Reply:
As @cwgoes pointed out, this is not `frequency` but a `period`. I you have no objections, I'd change it to `inv-checks-period` - naming suggestions are welcome though!

Reply:
Ahh yes, by all means!